org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
 {
-    "id": "org.musicbrainz.Picard",
+    "app-id": "org.musicbrainz.Picard",
```

Comment:
Please drop this change.

"id" and "app-id" are entirely equivalent, so there is no reason to change.

In addition, I was just told the following in #flatpak:

![screenshot from 2018-03-28 15-23-08](https://user-images.githubusercontent.com/272170/38031675-097cddd4-329c-11e8-9176-a0da1567117e.png)

So I'd actually prefer we use the newer name.